src/library_webgpu.js

```diff
@@ -1692,15 +1685,11 @@ var LibraryWebGPU = {
 
   wgpuCommandEncoderClearBuffer: function(encoderId, bufferId, {{{ defineI64Param('offset') }}}, {{{ defineI64Param('size') }}}) {
     var commandEncoder = WebGPU.mgrCommandEncoder.get(encoderId);
-    {{{ receiveI64ParamAsI32s('offset') }}}
-    {{{ receiveI64ParamAsI32s('size') }}}
+    {{{ receiveI64ParamAsI53('offset') }}}
+    {{{ receiveI64ParamAsI53('size') }}}
 
     var buffer = WebGPU.mgrBuffer.get(bufferId);
-    commandEncoder["clearBuffer"](
-      buffer,
-      convertI32PairToI53(offset_low, offset_high),
-      convertI32PairToI53(size_low, size_high)
-    );
+    commandEncoder["clearBuffer"](buffer, offset, size);
```

Comment:
This bug was already here, but this should be
```suggestion
    commandEncoder["clearBuffer"](buffer, offset, size == -1 ? undefined : size);
```